portal/code/company-portal/src/components/cax-responsibilities.tsx

```diff
@@ -142,7 +138,9 @@ export const ResponsibilitiesCax = ({
   } else {
     toast.error("Email or User Role empty.");
```

Comment:
i18n missing?

Reply:
will handle i18n in this https://jira.catena-x.net/browse/CPLP-597